app/src/utils/upload-file.ts

```diff
@@ -27,7 +27,7 @@ export async function uploadFile(
 	const server = useServerStore();
 	let notified = false;
 
-	if (server.info.uploads) {
+	if (server.info.uploads?.tusEnabled) {
```

Comment:
Let's be explicit here 
```suggestion
	if (server.info.uploads?.tusEnabled === true) {
```